app/controllers/images_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ImagesController < ApplicationController
+
+  # GET /images
+  # GET /images.json
+  def index
+    @images = Image.all
+
+    respond_to do |format|
+      format.html # index.html.erb
+      format.json { render json: @images }
```

Comment:
I think we don't need standard format closures but rather better to use respond_to :html and respond_with

gem called responders